Readability.js

```diff
@@ -1571,7 +1579,7 @@ Readability.prototype = {
     if (!metadata.excerpt) {
       var paragraphs = articleContent.getElementsByTagName("p");
       if (paragraphs.length > 0) {
```

Comment:
Huh, can you file a followup issue to make this less dumb? If we only care about the first P, we should just do our own iteration.